config/development_instance2.yaml

```diff
@@ -0,0 +1,145 @@
+#
+# this is a copy of development.yaml, but with different ports for the services,
+# and bootstrapping on development.yaml's service hosts.
+#
+# this is intended for running multiple instances locally, within the same cluster.
+# start `--env development` first, then `--env development_instance2`.
+#
+
+persistence:
+  defaultStore: cass-default
+  visibilityStore: cass-visibility
+  numHistoryShards: 4
+  datastores:
+    cass-default:
+      nosql:
+        pluginName: "cassandra"
+        hosts: "127.0.0.1"
+        keyspace: "cadence"
+        connectTimeout: 2s # defaults to 2s if not defined
+        timeout: 5s # defaults to 10s if not defined
+        consistency: LOCAL_QUORUM # default value
+        serialConsistency: LOCAL_SERIAL # default value
+    cass-visibility:
+      nosql:
+        pluginName: "cassandra"
+        hosts: "127.0.0.1"
+        keyspace: "cadence_visibility"
+
+ringpop:
+  name: cadence
+  bootstrapMode: hosts
+  bootstrapHosts: [
+    "127.0.0.1:7933", "127.0.0.1:7934", "127.0.0.1:7935",   # development.yaml list, for instance 1
+    "127.0.0.1:27933", "127.0.0.1:27934", "127.0.0.1:27935" # for this instance
+  ]
```

Comment:
this is the only change from `development.yaml` and....

tbh it kinda makes me think this file might not be worth it.  though documenting the strategy in `development.yaml` would be good.  
this didn't take long to figure out, but it was mostly a lucky guess rather than an "obviously this is correct" change.